test/integration_test/basic_test.go

```diff
@@ -325,16 +325,26 @@ func BasicInstallInCustomNamespace(tc *types.TestCase) func(*testing.T) {
 			err = ci_utils.ValidateObjectsAreTerminated([]runtime.Object{ns}, false)
 			require.NoError(t, err)
 		}
+
+		// Delete and validate StorageCluster deletion
+		ci_utils.UninstallAndValidateStorageCluster(cluster, t)
```

Comment:
Maybe do a BeforeEach/AfterEach here to ensure this happens?

Reply:
nevermind, discussed offline

Reply:
I think this won't work with the methods we have, let's test `px-wipe` after each test for all the different combinations we have